libbeat/common/transport/tlscommon/config.go

```diff
@@ -25,13 +25,13 @@ import (
 
 // Config defines the user configurable options in the yaml file.
 type Config struct {
-	Enabled          *bool                   `config:"enabled" yaml:"enabled"`
+	Enabled          *bool                   `config:"enabled" yaml:"enabled,omitempty"`
```

Comment:
I think we can change 'Enabled' to `bool`. The `IsEnabled` method also defaults 'Enabled' to true.